### ISSUE TITLE
Hovertool datetime shows as percentage. 

### ISSUE THREAD

**Issue by epa095** [PatrikHlobil/Pandas-Bokeh#60]
Code:
```python
import pandas as pd
import numpy as np

pd.set_option("plotting.backend", "pandas_bokeh")
np.random.seed(42)
df = pd.DataFrame({"Google": np.random.randn(1000)+0.2, 
                   "Apple": np.random.randn(1000)+0.17}, 
                   index=pd.date_range('1/1/2000', periods=1000))
df = df.cumsum()
df = df + 50

df.plot()
```
Result: 
![bukeh-example](https://user-images.githubusercontent.com/1576438/76428607-2dad1a80-63ae-11ea-858c-3bafa9801090.png)

Relevant versions:
bokeh==2.0.0
pandas==1.0.1
pandas-bokeh==0.4.2

**Comment by PatrikHlobil** [comment]
Hi @epa095,

Thanks for reporting this issue. I guess that happened with the release of bokeh 2.0. I have to look in the code to see what happens here.

Best Patrik

**Comment by PatrikHlobil** [comment]
Hi @epa095,

the problem was indeed related to Bokeh >= 2.0. With the Bokeh=0.5 Release the issue is fixed and Bokeh >= 2.0 is enforced. For now, please use Bokeh=1.4 if possible.

Best Patrik
